app/repositories/stage-configuration-repository.js

```diff
@@ -0,0 +1,27 @@
+const { models } = require('../data')
+/**
+ * Get stage configuration by id
+ * @param {integer} id
+ * @returns stage configuration object
+ */
+async function get (id) {
+  return models.stage_configuration.findAll(
+    {
+      where: { id }
+    })
+}
+
+/**
+ *
+ * @param {*} data
+ * @returns
+ */
+async function set (data) {
```

Comment:
This data will be inserted into the db table using a manual script, there is no need for a set function at this time.